### ISSUE TITLE
Finalize CSPICE build process

### ISSUE THREAD

**Issue by nicklafarge** [mbd-astro/forest#14]

**Comment by rjpower4** [comment]
Do we want to add CSPICE before we are working on the ephemeris integration? I thought we'd hold off until we are doing that so we don't carry around that dependency yet.

**Comment by nicklafarge** [comment]
Agreed - I'm trying to organize by milestone, so this is a 1.0 issue (i.e. we'll get to it in the Spring)
